cif_core.dic

```diff
@@ -16876,6 +16876,30 @@ save_DATABASE_CODE
 
 save_
 
+save_database_code.bincstrdb
+
+    _definition.id                '_database_code.BIncStrDB'
+
+    loop_
+      _alias.definition_id
+         '_database_code_BIncStrDB'
```

Comment:
If I understand correctly, B-IncStrDB maintainers will be the ones assigning these codes, so it would probably be best to consult with them if they plan to use this dotless alias in their data files. If they are fully contempt with only using the main dotted name, then I would probably remove this alias (it can always be added later on if needed).